modules/dolt/dolt.go

```diff
@@ -0,0 +1,225 @@
+package dolt
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"github.com/testcontainers/testcontainers-go/wait"
+	"path/filepath"
```

Comment:
You probably need to run `make lint` from the module dir 🙏 

I think the code generation tool does it for first time, but then there is no pre-commit rule to enforce it.

Reply:
done